src/Compilers/VisualBasic/Test/Emit/Emit/CompilationEmitTests.vb

```diff
@@ -1037,6 +1037,38 @@ End Class
 Sub(comp) comp.AssertTheseDiagnostics())
         End Sub
 
+        <Fact, WorkItem(49470, "https://github.com/dotnet/roslyn/issues/49470")>
+        Public Sub RefAssemblyClient_EventBackingField()
+            Dim lib_vb = "
+Imports System
+
+Public Class Button
+    Public Event Click As EventHandler
+End Class
+
+Public Class Base
+    Protected WithEvents Button1 As Button
+End Class
+"
+            Dim source_vb = "
+Imports System
+
+Public Class Derived
+    Inherits Base
+
+    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
+    End Sub
+End Class
+"
+            Dim libComp = CreateCompilationWithMscorlib40({Parse(lib_vb)}, options:=TestOptions.DebugDll.WithDeterministic(True))
```

Comment:
>WithMscorlib40 [](start = 43, length = 14)

Is "WithMscorlib40" necessary?

Reply:
It is necessary. 
Without it we get diagnostics about missing references in netstandard 2.0. This is following the pattern of existing ref assembly tests.

---
In reply to: [569596103](https://github.com/dotnet/roslyn/pull/50930#discussion_r569596103) [](ancestors = 569596103)